components/compass-runtime-agent/go.mod

```diff
@@ -87,5 +87,5 @@ replace (
 	github.com/containerd/containerd => github.com/containerd/containerd v1.4.11
 	github.com/docker/distribution => github.com/docker/distribution v0.0.0-20191216044856-a8371794149d
 	github.com/docker/docker => github.com/docker/docker v20.10.8+incompatible
-	github.com/kyma-project/kyma/components/application-operator => github.com/koala7659/kyma/components/application-operator v0.0.0-20211207101924-24d09934d9b6
```

Comment:
We have additional dependency to application-operator in compass-runtime-agent - we are using exported function to normalize imported API Bundle name. See components/compass-runtime-agent/internal/kyma/applications/converter.go line 142